nistats/first_level_model.py

```diff
@@ -444,8 +451,12 @@ def fit(self, run_imgs, events=None, confounds=None,
             # Mask and prepare data for GLM
             if self.verbose > 1:
                 t_masking = time.time()
-                sys.stderr.write('Starting masker computation')
-            Y = self.masker_.transform(run_img)
+                sys.stderr.write('Starting masker computation \r')
+
+            # ATTENTION THIS IS NOT SUPPOSED TO BE HERE
```

Comment:
Oh definitely im sorry, I forgot to swap back this line of code. It was a simple sanity check on the masker behavior but I forgot to revert it, good thing I commented on that. Also the import showed the same comment.